app/proto/admin.proto

```diff
@@ -268,3 +271,8 @@ message ListUserIdsRes {
 
   string next_page_token = 2;
 }
+
+message EditReferenceTextReq {
+  int64 reference_id = 1;
+  string new_text = 6; //plain text
```

Comment:
Please do `= 2` here. It's the gRPC tag number (grpc doesn't send the field name, just the identifier and type)